tabs/src/components/WalletAllowanceCard.tsx

```diff
@@ -34,16 +37,29 @@ const WalletAllowanceCard: React.FC<AllowanceCardProps> = ({
 
   const fetchAmountReceived = async () => {
     console.log('Fetching your wallet ...');
-    const walletForUser = await getUserWallets(adminKey, '2984e3ac627e4fea9fd6dde9c4df83b5');
 
-    console.log('Wallets: ', walletForUser);
+    const currentUserLNbitDetails = await getUsers(adminKey, {
```

Comment:
I don't think you need to do this. Just call `getUser`, then `currentUserLNbitsDetails.allowanceWallet.balance_msat`

Reply:
Interesting.. yes.. updated this for Private and Allowance wallet too.